src/Symfony/Component/Routing/Tests/Generator/UrlGeneratorTest.php

```diff
@@ -856,6 +856,23 @@ public function testFragmentsCanBeDefinedAsDefaults()
         $this->assertEquals('/app.php/testing#fragment', $url);
     }
 
+    public function testNonStringableObjectParameterCannotBeConvertedToString()
+    {
+        $this->expectException(InvalidParameterException::class);
+        $object = new \stdClass();
+        $routes = $this->getRoutes('test', new Route('/testing'));
```

Comment:
```suggestion
        $object = new \stdClass();
        $routes = $this->getRoutes('test', new Route('/testing'));
        $this->expectException(InvalidParameterException::class);
```

I had a discussion with @wouterj about this the other day. See https://github.com/symfony/symfony/pull/40284#discussion_r582939157

Reply:
I just tried to stay consistent with other uses of the same method in the file.
But I agree that expectation near statement is better.